deps/kms/src/plugins/sample.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use aes_gcm::{aead::Aead, Aes256Gcm, Key, KeyInit, Nonce};
+use anyhow::{anyhow, Result};
+use async_trait::async_trait;
+use crypto::WrapType;
+use rand::RngCore;
+use zeroize::Zeroizing;
+
+use crate::{Datakey as DatakeyTrait, KMS};
```

Comment:
The alias is a bit confusing, because you later define a `pub struct Datakey`. I suggest:

1. `use crate::{Datakey, KMS}`
2. line in 34: `pub struct SampleDatakey` (and then later: `impl Datakey for SampleDatakey`)

Reply:
Thanks for the suggestion. Fixed it